Meson linker errors when building statically: unresolved externals

Downloading the repo and running `meson build --default-library=static -Ddocs=false --backend=vs` and then building will result in linker errors. The same sort of issue results with the ninja backend as well. Full errors below.

```
1>------ Build started: Project: fribidi (bin\fribidi), Configuration: debug x64 ------
1>fribidi-main.c.obj : error LNK2019: unresolved external symbol __imp_fribidi_set_debug referenced in function main
1>fribidi-main.c.obj : error LNK2019: unresolved external symbol __imp_fribidi_get_bidi_type referenced in function main
1>fribidi-main.c.obj : error LNK2019: unresolved external symbol __imp_fribidi_charset_to_unicode referenced in function main
1>fribidi-main.c.obj : error LNK2019: unresolved external symbol __imp_fribidi_unicode_to_charset referenced in function main
1>fribidi-main.c.obj : error LNK2019: unresolved external symbol __imp_fribidi_parse_charset referenced in function main
1>fribidi-main.c.obj : error LNK2019: unresolved external symbol __imp_fribidi_char_set_name referenced in function help
1>fribidi-main.c.obj : error LNK2019: unresolved external symbol __imp_fribidi_char_set_title referenced in function help
1>fribidi-main.c.obj : error LNK2019: unresolved external symbol __imp_fribidi_char_set_desc referenced in function help
1>fribidi-main.c.obj : error LNK2019: unresolved external symbol __imp_fribidi_set_mirroring referenced in function main
1>fribidi-main.c.obj : error LNK2019: unresolved external symbol __imp_fribidi_set_reorder_nsm referenced in function main
1>fribidi-main.c.obj : error LNK2019: unresolved external symbol __imp_fribidi_remove_bidi_marks referenced in function main
1>fribidi-main.c.obj : error LNK2019: unresolved external symbol __imp_fribidi_log2vis referenced in function main
1>fribidi-main.c.obj : error LNK2001: unresolved external symbol __imp_fribidi_version_info
1>.\fribidi.exe : fatal error LNK1120: 13 unresolved externals
1>Done building project "51a1f05@@fribidi@exe.vcxproj" -- FAILED.
========== Build: 0 succeeded, 1 failed, 16 up-to-date, 0 skipped ==========
```

--- a/bin/meson.build
+++ b/bin/meson.build
@@ -1,27 +1,27 @@
 fribidi = executable('fribidi',
   'fribidi-main.c', 'getopt.c', 'getopt1.c', fribidi_unicode_version_h,
-  c_args: ['-DHAVE_CONFIG_H'],
+  c_args: ['-DHAVE_CONFIG_H'] + visibility_args,
   include_directories: incs,
   link_with: libfribidi,
   install: true)
 
 executable('fribidi-benchmark',
   'fribidi-benchmark.c', 'getopt.c', 'getopt1.c', fribidi_unicode_version_h,
-  c_args: ['-DHAVE_CONFIG_H'],
+  c_args: ['-DHAVE_CONFIG_H'] + visibility_args,
   include_directories: incs,
   link_with: libfribidi,
   install: false)
 
 executable('fribidi-bidi-types',
   'fribidi-bidi-types.c', fribidi_unicode_version_h,
-  c_args: ['-DHAVE_CONFIG_H'],
+  c_args: ['-DHAVE_CONFIG_H'] + visibility_args,
   include_directories: incs,
   link_with: libfribidi,
   install: false)
 
 executable('fribidi-caprtl2utf8',
   'fribidi-caprtl2utf8.c', fribidi_unicode_version_h,
-  c_args: ['-DHAVE_CONFIG_H'],
+  c_args: ['-DHAVE_CONFIG_H'] + visibility_args,
   include_directories: incs,
   link_with: libfribidi,
   install: false)

